Source/DafnyCore/DafnyCore.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Target Name="RunCoco" BeforeTargets="PreBuildEvent" Outputs="$(ProjectDir)Parser.cs;$(ProjectDir)Scanner.cs" Inputs="$(ProjectDir)Dafny.atg">
+  <Target Name="RunCoco" BeforeTargets="PreBuildEvent" Outputs="$(ProjectDir)Parser.cs;$(ProjectDir)Scanner.cs" Inputs="$(ProjectDir)Dafny.atg;$(ProjectDir)/CoCo/Parser.frame;$(ProjectDir)/CoCo/Scanner.frame">
```

Comment:
Ahhhhh that's why it was not reconstructing when I was changing Scanner.frame ! Can you please also add Parser.frame in inputs and Parser.cs in outputs?

Reply:
They are there already. But yes, I was very pleased when I found this was the only change needed fo this problem. The have been may times when I altered a .frame file and wondered why nothing changed in the program behavior.